treeherder/webapp/api/runnable_jobs.py

```diff
@@ -16,6 +20,21 @@ def list(self, request, project):
         """
         GET method implementation for list of all runnable buildbot jobs
         """
+        tc_jobs_url = request.query_params['tcURL']
```

Comment:
It seems odd to pass a url in to a url.  I wonder if there's just an ID that could be passed in.  This isn't a deal-breaker or anything, though